sdk/python/tests/dsl/main.py

```diff
@@ -20,14 +20,15 @@
 import pipeline_param_tests
 import container_op_tests
 import ops_group_tests
-
+import type_tests
 
 if __name__ == '__main__':
   suite = unittest.TestSuite()
   suite.addTests(unittest.defaultTestLoader.loadTestsFromModule(pipeline_param_tests))
   suite.addTests(unittest.defaultTestLoader.loadTestsFromModule(pipeline_tests))
   suite.addTests(unittest.defaultTestLoader.loadTestsFromModule(container_op_tests))
   suite.addTests(unittest.defaultTestLoader.loadTestsFromModule(ops_group_tests))
+  suite.addTests(unittest.defaultTestLoader.loadTestsFromModule(type_tests))
```

Comment:
JFYI: the `unittest` module has buildtin test discovery. By default its search pattern is `test*.py`. See https://docs.python.org/3/library/unittest.html#unittest.TestLoader.discover https://docs.python.org/3/library/unittest.html#test-discovery

Reply:
thanks